### PR TITLE
add basic formating of params for "gnocchi archive-policy update"

### PR DESCRIPTION
**Pull request by gujun4990**
When updating a archive-policy using gnocchi command and the definition include space character, the exception is raised. The command is like this: gnocchi archive-policy update -d "granularity: 0:03:00, points: 86400" low. So the commit will fix the bug.

### CODE REVIEW

File: gnocchiclient/v1/archive_policy_cli.py [hunk 1]
```diff
@@ -54,6 +54,8 @@ def archive_policy_definition(string):
     defs = {}
     for part in parts:
         attr, __, value = part.partition(":")
+        attr = attr.replace(' ', '')
+        value = value.replace(' ', '')
```
Comment (chungg) [inline gnocchiclient/v1/archive_policy_cli.py]: i think you can put `parts = string.replace(' ', '').split(",")` in line 53 so you just need  to run replace once and be done.

Comment (gujun4990) [reply gnocchiclient/v1/archive_policy_cli.py]: Thank you for your suggestion. But I found the travis CI build failed. I analyzed the test code and can be run, so I don't know the cause of the failure.

**Comment by chungg** [comment]
strange, there seems to be some unrelated pep8 issues.

in the meantime, i'd suggest you fix you the formatting of your commit message so the first line is the title, and the 3rd line onwards describes your PR.

also, you should try to squash your PR in to one commit rather than appending changes as new commits. i'm pretty sure there's other ways to do this but i use `git rebase -i HEAD~3` and you squash the ones you need to.

**Comment by gujun4990** [comment]
Thanks for your reminder. This helps me a lot. I have merged one commit and modified the commit message. Please merge it if having no error. Thank you.

**Comment by jd** [comment]
Not sure it's worth a change. There's a syntax, I don't see why we should make it accept more stuff?

**Pull request closed without merging**